### ISSUE TITLE
vkDeferredOperationJoinKHR crashes in AMD GPU

### ISSUE THREAD

**Issue by locke-lunarg** [KhronosGroup/Vulkan-ValidationLayers#3825]
- OS: Win 11
 - GPU: AMD RX 6800 XT. Driver: 22.2.1

If I enabled ValidationLayers, vkDeferredOperationJoinKHR will crash if vkCreateRayTracingPipelinesKHR uses VkDeferredOperationKHR. The reason might be that according to vkspec
```
Parameters to the command requesting a deferred operation may be accessed at any time until the deferred operation enters the pending state. While a deferred operation is pending:

Externally synchronized parameters must not be accessed.

Pointer parameters must not be modified (e.g. reallocated/freed).

The contents of pointer parameters which may be read by the command must not be modified.

The contents of pointer parameters which may be written by the command must not be read.

Vulkan object parameters must not be passed as externally synchronized parameters to any other command.
```
ValidationLayers wraps pipeline handles in vkCreateRayTracingPipelinesKHR to cause vkDeferredOperationJoinKHR crash. It has to add `VK_VALIDATION_FEATURE_DISABLE_UNIQUE_HANDLES_EXT` in `VkInstanceCreateInfo::pNext` to disable wrapping handles. My Nvidia GPU doesn't crash for that. I'm not sure if it should be count as a bug.

**Comment by aitor-lunarg** [comment]
I think this issue may be fixed with #3793

**Comment by locke-lunarg** [comment]
Sound good. I will test it.
